README.md

```diff
@@ -41,38 +80,221 @@ Checkout [architectures notes](doc/Architecture.md)
   - `tools/packets`: generate packets structure from database file
   - `tools/skills`: generate skills structure from configuration file 
 
-# TODO
-Focus is now on [Skills](https://github.com/nmeylan/rust-ro/issues/11)
 
-Also see [Meta issue](https://github.com/nmeylan/rust-ro/issues/19)
+## 5. Setup 
 
-# Usage
-*This section will be rewritten*
+Here's a list of **pre-requisites** to run rust-ro:
 
-**Pre-requisite:**
-- postgresql (rathena db was converted using pgloader and then adapted)
-- RO db `db/pg.sql`
+* Docker OR PostgreSQL 16+ directly on your machine 
+* Rust - nighly build
 
-**Notes:**
-- All packets for account 2000000 are handle by this project.
-- All packets for any other account are proxied (and display in console) to hercules or rathena.
+### 5.1 Config
+
+First, make a copy from `config.template.json` to `config.json`:
+
+```shell
+cd rust-ro
+cp config.template.json config.json
+```
+Inside this JSON, you will find **database related variables** and **game related variables** (exp_rate, drop_rate etc) as well. You can change in the way you want, but for now let's leave it with default values.
+
+```json
+// config.json
+[
+  "game": {
+    "default_char_speed": 150,
+    "drop_rate": 1.0,
+    "drop_rate_card": 1.0,
+    "drop_rate_mvp": 1.0,
+    "max_base_level": 99,
+    "max_inventory": 100,
+    "mob_density": 1.0,
+    "max_stat_level": 99,
+    "base_exp_rate": 1.0,
+    "job_exp_rate": 1.0,
+    "mob_dropped_item_locked_to_owner_duration_in_secs": 30,
+    "player_dropped_item_locked_to_owner_duration_in_secs": 60
+  }
+]
+```
+
+> TODO: make a small info for each variable, maybe a wiki?
+
+### 5.2 Setup DB
+
+The entire database structure was based on **rAthena** but instead of using MySQL, we decided to go with PostgreSQL. There's minor modifications so far but until we mapped some **constraints**. 
+
+Since there's features like `ON CONFLICT (column) .. DO UPDATE`, `UNNEST(array)` that is not provided by MySQL, PostgreSQL took place to leverage the project in the database aspect.
+
+#### 5.2.1 Setup DB: Docker
+
+If you already have **Docker** installed in your machine, we prepared a **docker-compose.yml** with all configs ready for your ragnarok server.
+
+Go to */docker* and run:
+```shell
+docker-compose up -d
+```
+
+By default, we ran `/rust-ro/docker/volumes/init.sql`, script that creates a database, user and grant all privileges to the user `ragnarok`. It also copies the `/rust-ro/docker/volumes/pg.sql` volume to your docker instance.
+
+```sql
+-- In case you want to take a look: 
+-- /rust-ro/docker/volumes/init.sql
+
+CREATE USER ragnarok WITH PASSWORD 'ragnarok';
+CREATE DATABASE ragnarok;
+GRANT ALL PRIVILEGES ON DATABASE ragnarok TO ragnarok;
+ALTER DATABASE ragnarok OWNER TO ragnarok;
+```
+
+After your database is up, you will need import the Game SQL (`/rust-ro/pg.sql`) to our new database. For that follow the steps:
```

Comment:
normally all step below should be automated, if it is not the case we can create an issue for this, but leaving your input for now

Reply:
When I tried to install it didn't worked as expected. Maybe missed something? IDK but we can improve this process later :D